build/lib/builder.js

```diff
@@ -148,6 +170,17 @@ class Builder {
 		await fs.remove(TMP_COMMON_DIR);
 	}
 
+	async transpile(platform, babelOptions, outDir) {
+		if (!outDir) {
+			outDir = path.join(TMP_DIR, 'common', platform);
+		}
+		await fs.emptyDir(outDir);
+		return Promise.all([
+			this.generateTiMain(platform, babelOptions, outDir),
+			this.generateKernelBundle(platform, babelOptions, outDir),
+		]);
+	}
+
```

Comment:
Also, we don't really need "/common/Resources/android/ti.kernel.js" to be packaged with the SDK for Android since this JS file is embedded in the C/C++ side. I verified this "ti.kernel.js" is being added as an asset within the APK. It's only about 200kb and not a big deal if bundled within the Android app, but it isn't necessary either.